src/redux/store.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
+import { configureStore } from '@reduxjs/toolkit';
+import categoriesReducer from './categories/categoriesSlice';
+import bookReducer from './books/bookSlice';
+/* eslint-enable */
+
```

Comment:
- [ ] Please always implement the things instructed by your code reviewer. I can see that after getting the review, you still haven't removed the `eslinter` disable and enable comment. As I tried at my end, there is no error after removing it. So please remove these comments in order to get project approval.